docs/dashboards/filter-template-variables.md

```diff
@@ -76,6 +80,29 @@ To add a Logs Search variable to a dashboard, do the following:
 1. Select **Logs Search** as the **Variable Type**.
 1. Click in the **Query** field and begin typing your query. Select valid options from the pop-up list as they appear until your query is complete.
 1. The **Key** is the metadata field you want to use as the filter. Once a Key is selected the Preview table will show example values based on the query running for the last 15 minutes.
+1. (Optional) **Include the option to select all values (\*)** will be selected by default. This includes a wildcard asterisk (\*) in the available options.
+1. (Optional) Enter a **Default Value** for the variable.
+1. Click **Create Template Variable** to apply the variable to the dashboard.
+
+### Add a Scope-based Variable 
+
+With Scope-based variables, you can define a variable automatically applies to all log queries within the scope of the dashboard or panels. With this you can define the key while creating the scope variable and users can select the appropriate value to implement simple filtering for the dashboard. Additionally with Scope-based variables, you can select the specific panels to which the scope variable should be applied.
```

Comment:
I would make the casing consistent for "scope-based variable(s)" throughout the doc